projects/table-builder/src/lib/components/initialization-component/initialization-component.html

```diff
@@ -31,6 +31,9 @@
 <ng-template #expression let-element='element' let-metaData='metaData'>
   <span>{{element | func : metaData.transform }}</span>
 </ng-template>
+<ng-template #enum let-element='element' let-metaData='metaData' let-transform='transform'>
+  <span>{{ metaData.additional.enumMap[element[metaData.key]] }}</span>
```

Comment:
did you forget to add the func pipe

Reply:
Not really. why do I need the func pipe?